main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -67,40 +102,50 @@ chainHub.registerConnection(
 );
 ```
 
-In this example, `chainHub` is used to register a new chain and establish a connection between the Agoric chain and the newly registered chain.
+In this example, `chainHub` is used to register a new chain and establish a connection 
+between the Agoric chain and the newly registered chain.
 
-### Orchestration Account
+## Interchain Account (ICA)
```

Comment:
Also, please maket his section consistent by adding APIs subsections. Also currently it seems to be vague and marketing speak. For example, it is not clear to me at first what is ICA after all in the code example bellow. 
Are omniAccount and localAccount ICA objects? If yes, then can you add getAddress, delegate, and transferSteps as API subsections in this ICA section?

Reply:
I was thinking of removing this subsection but kept it for now. I don't understand it well enough to be certain, but I believe we do not use the ICA in the code that is externally accessible to the developers. Rather we use the orchestration account that I already explain above.